src/components/Editor/Editor.css

```diff
@@ -1,8 +1,12 @@
-/* vim:set ts=2 sw=2 sts=2 et: */
+.editor-wrapper {
+  --debug-line-background: rgba(226, 236, 247, 0.5);
+  --debug-line-border: rgb(145, 188, 219);
+}
 
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
that's what i was thinking, but I'd like to go through and add them to all the files and it's easier to be consistent :)